src/utils/FirebaseLib.js

```diff
@@ -27,6 +27,15 @@ function fireBaseWrapper(){
   this.dataOn = function(eventType, callBack){
     this.fireBase.on(eventType, callBack);
   };
+
+  this.getEloDateForCurrentMonth = function(playerId, eventType, callBack){
+    let date = [new Date().getFullYear(), new Date().getMonth()].join('_');
+
```

Comment:
Maybe cache the date object.
`let date = [new Date().getFullYear(), new Date().getMonth()].join('_');`
to 
```js
let now = new Date();
let date = [now.getFullYear(), now.getMonth()].join('_');
```